test/specs/simulation.vehicles.spec.js

```diff
@@ -1,19 +1,37 @@
-const { generateRandomVehicles } = require('../../server/simulation/vehicles.js');
+const {generateRandomVehicles, randomBid} = require('../../server/simulation/vehicles.js');
```

Comment:
There are a lot of formatting changes introduced in this file that have nothing to do with what you added.
My guess is one of your tools (prettier?) introduced them automatically.

When contributing to open source, try to follow the existing code style of the project you are contributing to (in this example it is spaces around the variable names in line 1), and only change the relevant code.

If you'd like to introduce further unrelated changes, you can do them in a separate pull request